edi_oca/models/edi_exchange_record.py

```diff
@@ -130,7 +130,7 @@ def _compute_related_name(self):
             related_record = rec.record
             rec.related_name = related_record.display_name if related_record else ""
 
-    @api.depends("model", "type_id")
+    @api.depends("model", "res_id", "type_id")
```

Comment:
If it is not possible to duplicate then I think there is no need to recalculate when **`res_id`** changes
```suggestion
    @api.depends("model", "type_id")
```